### PR TITLE
Design for the attachment count component

### PR DESCRIPTION
**Pull request by mborsare**
This PR extends an existing component to better fit into its context. It
also DRYs up some of the border declarations throughout.

attachments

https://trello.com/c/tSoQIx2O

### CODE REVIEW

**Comment by alexandra-aas** [comment]
From a visual perspective, I wonder if on hover it would look a bit cleaner if the underline just went under the text and then the  background of the count bubble changed to maroon.

File: app/assets/stylesheets/components/_split-tabs.scss [hunk 1]
```diff
@@ -45,3 +45,9 @@
     background-color: $mit-red;
   }
 }
+
+.emphasized-tab-digit-small {
```
Comment (alexandra-aas) [inline app/assets/stylesheets/components/_split-tabs.scss]: If this is a modified of `.split-tab-digit`, I would nest it inside `.split-tab-digit` to show that it needs to be coupled with it to work.

Comment (alexandra-aas) [inline app/assets/stylesheets/components/_split-tabs.scss]: If this is a modified of `.split-tab-digit`, I would nest it inside `.split-tab-digit` to show that it needs to be coupled with it to work.

**Comment by mborsare** [comment]
Good idea! I'll give it a shot.
```
From a visual perspective, I wonder if on hover it would look a bit cleaner if the underline just went under the text and then the background of the count bubble changed to maroon
```

**Comment by mborsare** [comment]
attachments-clean

**Pull request merged**